src/components/Navbar/Navbar.js

```diff
@@ -119,7 +119,11 @@ const MobileIcon = styled.div`
 `;
 
 const Navbar = ({ toggle }) => {
-  const { data: session } = useSession();
+  const { data: session, status } = useSession();
+  var isAuth = false;
+  if (status === "authenicated") {
+    isAuth = true;
+  }
```

Comment:
Please use `const` instead of `var`. This can also be simplified into a single liner:

`const isAuth = status === "authenicated"`;